scripts/staging-ci/staging-reset.js

```diff
@@ -52,6 +59,13 @@ async function main() {
   }
 
   printLog('Reset done.')
+
+  const commitMessage = await executeCommand('git show-branch --no-name HEAD')
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏the HEAD commit message is not the one that triggered the pipeline. It'll be probably the last commit done in this script, `Remove ${CI_FILE} on old branch so pushes are noop`

But I find it a bit weird to have the commit message in scheduled tasks, no? WDYT? 

(Same for the failure message and the chrome driver bump job)

Reply:
Yes right. I replaced the commit message by the pipeline id for scheduled tasks